components/rmrk/Profile/NavbarProfileDropdown.vue

```diff
@@ -64,9 +59,7 @@
       <b-dropdown-item custom aria-role="menuitem">
         <b-field>
           <p class="control">
-            <b-button type="is-primary" @click="changeAccount = !changeAccount">
-              Change account
-            </b-button>
+            <ConnectWalletButton buttonText="Change account" />
```

Comment:
```suggestion
            <ConnectWalletButton buttonText="general.account.change" />
```

add translation pls 🥺